qiskit/circuit/controlflow/if_else.py

```diff
@@ -205,7 +209,11 @@ def __init__(
         self.__false_block: Optional[ControlFlowBuilderBlock] = false_block
         self.__resources = self._calculate_placeholder_resources()
         super().__init__(
-            "if_else", len(self.__resources.qubits), len(self.__resources.clbits), [], label=label
+            "if_else",
+            len(self.__resources.qubits),
+            len(self.__resources.clbits),
+            [],
+            label=label,
```

Comment:
I think you might have your `black` set to default to too-narrow line length.  Can you delete the trailing comma on `label=label` here (and everywhere else relevant) so the formatting goes back to what it was?